ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/rewrite/testtool/Cli.scala

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.rewrite.testtool
+
+import java.io.File
+
+import com.digitalasset.ledger.api.tls.TlsConfiguration
+
+object Cli {
```

Comment:
Yes, I initially copied and pasted the code to avoid merge errors. The work I'm doing right now consists in swapping the two implementations, which luckily had no conflicts.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2282/commits/03dd37dd168713526d000581eec659ce8767e734 (WARNING: it's a **large** commit)